packages/flutter/test/material/ink_sparkle_test.dart

```diff
@@ -140,7 +140,7 @@ Future<void> _runTest(WidgetTester tester, String positionName, double distanceF
           key: repaintKey,
           child: ElevatedButton(
             key: buttonKey,
-            style: ElevatedButton.styleFrom(splashFactory: InkSparkle.constantTurbulenceSeedSplashFactory),
```

Comment:
I figured this should be the best validator, since these goldens will fail otherwise. Easy test!

Reply:
Oh I think this is the wrong one. 😆